features/net/FEATURE_IPV4/lwip-interface/lwip-eth/arch/TARGET_NUVOTON/TARGET_NUC472/nuc472_eth.c

```diff
@@ -252,15 +252,15 @@ void  ETH_halt(void)
     EMAC->CTL &= ~(EMAC_CTL_RXON_Msk | EMAC_CTL_TXON_Msk);
 }
 
-
+unsigned int m_status;
```

Comment:
`#define     __IO    volatile`
All fields of EMAC are volatile, for ex: ` __IO uint32_t INTSTS;`
So, `m_status = EMAC->INTSTS & 0xFFFF; `should be OK.